src/plugins/kibana_utils/public/render_complete/render_complete_dispatcher.ts

```diff
@@ -64,7 +64,6 @@ export class RenderCompleteDispatcher {
     if (!this.el) return;
     this.count++;
     this.el.setAttribute('data-render-complete', 'true');
-    this.el.setAttribute('data-loading', 'false');
     this.el.setAttribute('data-rendering-count', String(this.count));
```

Comment:
The `waitForRender` method expects no `data-loading` attribute **at all** for an element to be considered rendered for functional tests - that is, `data-loading=false` is still considered to be a loading element. Therefore, we should not be setting `data-loading` to `false` here, because otherwise panels with errors are never considered to be done rendering by this test: 

https://github.com/elastic/kibana/blob/b1d9e9aca6c9f830081a5a35aadbe3ff4e00fd82/test/functional/services/renderable.ts#LL27C65-L27C65